index.html

```diff
@@ -266,6 +265,7 @@
       </div>
 
     </div>
+    <script src="dist/htmlminifier.min.js"></script>
```

Comment:
Maybe squash this commit with https://github.com/kangax/html-minifier/pull/569/commits/9f37bb689faef3501e6a5ae5b388047110c45017?